CONTRIBUTING.md

```diff
@@ -91,6 +91,10 @@ Make sure you have [Gradle](http://gradle.org) installed, as
 Elasticsearch uses it as its build system. Gradle must be at least
 version 3.3 in order to build successfully.
 
+We support development in the Eclipse and IntelliJ IDEs. For Eclipse, the
```

Comment:
Yes, we should say "support". If contributors/users have trouble in one of these two IDEs, then we as developers will help the, and we as developers will ensure that development in Elasticsearch can be done in one of these two IDEs. If they try to use any other IDE (e.g., Netbeans) we will not offer support. This encompasses the definition of "support" IMO.